cheatsheets/Protect_FileUpload_Against_Malicious_File.md

```diff
@@ -1,597 +1,138 @@
-                try (OutputStream fos = Files.newOutputStream(f.toPath(), StandardOpenOption.WRITE)) {
-                    if (!fallbackOnApacheCommonsImaging) {
-                        ImageIO.write(sanitizedImage, formatName, fos);
-                    } else {
-                        ImageParser imageParser;
-                        //Handle only formats for which Apache Commons Imaging can successfully write 
-                        // (YES in Write column of the reference link) the image format
-                        //See reference link in the class header
-                        switch (formatName) {
-                            case "TIFF": {
-                                imageParser = new TiffImageParser();
-                                break;
-                            }
-                            case "PCX": {
-                                imageParser = new PcxImageParser();
-                                break;
-                            }
-                            case "DCX": {
-                                imageParser = new DcxImageParser();
-                                break;
-                            }
-                            case "BMP": {
-                                imageParser = new BmpImageParser();
-                                break;
-                            }
-                            case "GIF": {
-                                imageParser = new GifImageParser();
-                                break;
-                            }
-                            case "PNG": {
-                                imageParser = new PngImageParser();
-                                break;
-                            }
-                            case "WBMP": {
-                                imageParser = new WbmpImageParser();
-                                break;
-                            }
-                            case "XBM": {
-                                imageParser = new XbmImageParser();
-                                break;
-                            }
-                            case "XPM": {
-                                imageParser = new XpmImageParser();
-                                break;
-                            }
-                            default: {
-                                throw new IOException("Format of the original image is not" + 
-                                                      " supported for write operation !");
-                            }
-
-                        }
-                        imageParser.writeImage(sanitizedImage, fos, new HashMap<>());
-                    }
-
-                }
-
-                // Set state flag
-                safeState = true;
-            }
-        } catch (Exception e) {
-            safeState = false;
-            LOG.warn("Error during Image file processing !", e);
-        }
-
-        return safeState;
-    }
-}
+```python
```

Comment:
Not sure this really adds much - it's very simple and we don't want to make it easier for people to blacklist. I'd say to remove this.